lib/setupmanagers/hckclient.rb

```diff
@@ -55,7 +55,13 @@ def restart_machine
       @tools.restart_machine(@name)
     end
 
+    def reconfigure_pool
+      @pool = @project.engine_tag
+    end
```

Comment:
Having `@pool = 'Default Pool'` does no longer make sense in the first place. Let's remove `@pool` and instead create a method `pool` that queries and returns the pool.